homeassistant/components/zwave/climate.py

```diff
@@ -256,6 +256,11 @@ def hvac_action(self):
         """
         return self._hvac_action
 
+    @property
+    def fan_state(self):
```

Comment:
Where is this used?

Reply:
I think we can remove this property and use `self._fan_state` directly in the state attributes.